### ISSUE TITLE
Broken Logging

### ISSUE THREAD

**Issue by MarvinTeichmann** [TensorVision/TensorVision#21]
Commit  47d4275 does destroy logging. After the commit no logging output is printed to the screen. I did not managed to find the issue. However I do consider this as urgent, as I do not want to merge before this is fixed, and I would like to build code on top of those commits.

**Comment by MartinThoma** [comment]
I'm not sure why, but

```
if 'TV_IS_DEV' in os.environ and os.environ['TV_IS_DEV']:
    logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s',
                        level=logging.DEBUG,
                        stream=sys.stdout)
else:
    logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s',
                        level=logging.INFO,
                        stream=sys.stdout)
```

has to be before `import tensorvision.utils as utils`

**Comment by MarvinTeichmann** [comment]
Ok, works. In future it would be good to test whether `tv-train` runs as expected. Do you know a way to automate this test? I.e. Run `tv-train --hypes examples/cifar10_vgg8.json` and check whether `2016-05-06 10:44:08,743 INFO Step 0/100000: loss = 4.99 ( 0.031 sec (per Batch); 4191.1 examples/sec)` appears?
